docs/csharp/programming-guide/concepts/async/cancel-async-tasks-after-a-period-of-time.md

```diff
@@ -4,213 +4,213 @@ ms.date: 07/20/2015
-                // ***Set up the CancellationTokenSource to cancel after 2.5 seconds. (You  
-                // can adjust the time.)  
-                cts.CancelAfter(2500);  
-  
-                await AccessTheWebAsync(cts.Token);  
-                resultsTextBox.Text += "\r\nDownloads succeeded.\r\n";  
-            }  
-            catch (OperationCanceledException)  
-            {  
-                resultsTextBox.Text += "\r\nDownloads canceled.\r\n";  
-            }  
-            catch (Exception)  
-            {  
-                resultsTextBox.Text += "\r\nDownloads failed.\r\n";  
-            }  
-  
-            cts = null;   
-        }  
-  
-        // You can still include a Cancel button if you want to.  
-        private void cancelButton_Click(object sender, RoutedEventArgs e)  
-        {  
-            if (cts != null)  
-            {  
-                cts.Cancel();  
-            }  
-        }  
-  
-        async Task AccessTheWebAsync(CancellationToken ct)  
-        {  
-            // Declare an HttpClient object.  
-            HttpClient client = new HttpClient();  
-  
-            // Make a list of web addresses.  
-            List<string> urlList = SetUpURLList();  
-  
-            foreach (var url in urlList)  
-            {  
-                // GetAsync returns a Task<HttpResponseMessage>.   
-                // Argument ct carries the message if the Cancel button is chosen.   
-                // Note that the Cancel button cancels all remaining downloads.  
-                HttpResponseMessage response = await client.GetAsync(url, ct);  
-  
-                // Retrieve the website contents from the HttpResponseMessage.  
-                byte[] urlContents = await response.Content.ReadAsByteArrayAsync();  
-  
-                resultsTextBox.Text +=  
-                    String.Format("\r\nLength of the downloaded string: {0}.\r\n", urlContents.Length);  
-            }  
-        }  
-  
-        private List<string> SetUpURLList()  
-        {  
-            List<string> urls = new List<string>   
-            {   
-                "http://msdn.microsoft.com",  
-                "http://msdn.microsoft.com/library/windows/apps/br211380.aspx",  
-                "http://msdn.microsoft.com/library/hh290136.aspx",  
-                "http://msdn.microsoft.com/library/ee256749.aspx",  
-                "http://msdn.microsoft.com/library/ms404677.aspx",  
-                "http://msdn.microsoft.com/library/ff730837.aspx"  
-            };  
-            return urls;  
-        }  
-    }  
-  
-    // Sample Output:  
-  
-    // Length of the downloaded string: 35990.  
-  
-    // Length of the downloaded string: 407399.  
-  
-    // Length of the downloaded string: 226091.  
-  
-    // Downloads canceled.  
-}  
-```  
-  
+> To run the examples, you must have Visual Studio 2012 or newer and the .NET Framework 4.5 or newer installed on your computer.
+
+## Downloading the Example
```

Comment:
This command also applies to all files.

nit: "Download" instead of "Downloading" (Several headings in all these files use gerund forms in headings.

related nit: Consider sentence case instead of Title Case for headings.